### DISCUSSION TITLE
Download videos from videopress.com

### DISCUSSION THREAD

**Discussion by krystobal123** [aclap-dev/video-downloadhelper#976] [Help and Q&A]
### Your question

Hello everyone.

I use a MacBook Pro. I installed the application on Firefox. I have installed the Companion application (v2.0.19).

I can't download videos from a site. As you can see from the screenshots, I have four suggestions. But three of them create an error. And the fourth only downloads the audio.

For other videos in the same series, I only have the image, but not the sound.

I've also taken a screenshot of the information in the video frame.

Have a nice day.

Krystof

![Capture d’écran 2024-07-05 à 19 57 17](https://github.com/aclap-dev/video-downloadhelper/assets/174827035/d8a05dcf-4cb5-49d4-b834-828856492add)

![PhotoScape X 2024-07-05 at 8 21 17 PM](https://github.com/aclap-dev/video-downloadhelper/assets/174827035/1be36695-6430-4dc9-b9ed-c946467ba235)

![Capture d’écran 2024-07-05 à 19 57 38](https://github.com/aclap-dev/video-downloadhelper/assets/174827035/e77e3b2b-9fbf-4a38-ac86-23e35b0fdd21)

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: fr
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:127.0) Gecko/20100101 Firefox/127.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "concurrent_downloads_max": 10,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": true,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": true
    },
    "yt_warning": false,
    "use_wide_ui": true,
    "use_sidebar": false,
    "open_count_store": 38,
    "successfull_dl": 9
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

One critical piece of information for helping other users is the URL of the web page on which you're having problems.  If this is a private site, paid or not, that requires a registration, then you're not going to get much help.  Why did you censor part of the URL?  We need that to help you.  Provide it in plain text within an ordinary post here & maybe somebody can help you.

**Reply by GrampaWildWilly** [reply]
I went to the home page of videopress.  There isn't much there, not even a search field that I could enter your video name in to try to find it.  There was one 35-second sample video on the page.  VDH downloaded it without problems.  But when I played it in VLC, it was silent.  I confirmed this 2 ways.  First, I played the video in the player on the web page.  It was silent.  I adjusted the volume control in the player to maximum & I turned up my computer volume while it was playing.  It was silent.  I then looked at the Hit Details in VDH.  It showed repeatedly that there was no audio track.

I don't know what all of this means for actual content within the site.  Maybe when you have trouble with something you download, you should play it in the player on the web page.  My test here shows that they can post content that really does not have an audio track.  Can.  Not necessarily always.  But possible.  But this looks to me very much like this is a pay site so I don't hold out much hope that you're going to get any useful help here.

**Comment by krystof1492** [comment]
Hello and thank you for your message.

Yes, it's a paid course (which I paid for).

I've listened to the course, but I'd like to download it locally onto my computer for greater convenience, and also to retrieve the audio that I'd like to turn into text so that I can study it at my leisure and keep a written paper copy.

I've hidden part of the URL precisely because it's a paid course.

I'll see if I can contact the author of these videos, and if he replies...

Thanks for your help, and have a great day.
